src/main/scala/uk/gov/hmrc/SbtAutoBuildPlugin.scala

```diff
@@ -168,8 +169,8 @@ object Git {
     }
   }
 
-  private def remoteUrl(remoteName: String): String = {
-    val remoteUrl = gitConfig.getString("remote", remoteName, "url")
+  private def remoteUrl(remoteName: String): Option[String] = {
```

Comment:
See above comment 

> I considered adding an additional test for this case, but it seems that the Git object needs significant restructuring before it can example files injected into it